_posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "This week in RustFest 6: CFP finalisation"
+authors:
+  - dns2utf8
+---
+
+This week we finalised the CFP. It was amazing to see many quality proposals from a variety of areas, disciplines, and backgrounds. This weekend we spent hours trying to put together a topic spanning conference lineup building on topics we haven't covered at Rustfest, as well as talks featuring the evolution of the story we continue to follow in topics we have visited in previous Rustfests. We're excited to send out mails to new speakers as well as returning speakers.
```

Comment:
The second sentence "This weekend ... previous RustFests" is quite long and confusing. How about the following:

This weekend we spent hours reviewing proposals and putting together the conference lineup. We did our best to cover both topics we yet haven't seen at Rustfest, as well as talks featuring the evolution of topics we have previously visited.